krop/src/main/java/com/avito/android/krop/KropView.kt

```diff
@@ -39,6 +40,7 @@ class KropView(context: Context, attrs: AttributeSet) : FrameLayout(context, att
                 offset = getDimensionPixelOffset(R.styleable.KropView_krop_offset, offset)
                 aspectX = getInteger(R.styleable.KropView_krop_aspectX, aspectX)
                 aspectY = getInteger(R.styleable.KropView_krop_aspectY, aspectY)
+                overlayShape = getInteger(R.styleable.KropView_krop_shape, SHAPE_OVAL)
```

Comment:
Already defined `overlayShape` may be used as default value here instead of `SHAPE_OVAL`.